synopsys-task/dist/index.js

```diff
@@ -952,7 +952,7 @@ exports.AZURE_ENVIRONMENT_VARIABLES = {
     AZURE_REPOSITORY: "Build.Repository.Name",
     AZURE_SOURCE_BRANCH: "Build.SourceBranch",
     AZURE_PULL_REQUEST_NUMBER: "System.PullRequest.PullRequestId",
-    AZURE_PULL_REQUEST_TARGET_BRANCH: "System.PullRequest.targetBranchName",
+    AZURE_PULL_REQUEST_TARGET_BRANCH: "System.PullRequest.TargetBranch",
```

Comment:
is there any issue with System.PullRequest.TargetBranch ? This also returns correct value.

Reply:
@spurohitsynopsys 
From the pipeline what I observed is, suppose for a feature branch named **dev/test/feature**
1. `Build.SourceBranch` returns **refs/heads/dev/test/feature**
2. `Build.SourceBranchName` returns **feature**
3. `System.PullRequest.TargetBranch` returns **refs/heads/dev/test/feature**
4. `System.PullRequest.targetBranchName` returns **dev/test/feature**

So if you see `SourceBranch` and `SourceBranchName` have different behavior. `SourceBranchName` trims of the hierarchical part too. So what I thought is maybe in future they can change the behavior of `targetBranchName` too, like it may return only the **feature** part. Therefore, for safety I decided too use `TargetBranch` instead of `targetBranchName`.

Let me know if it is okay or do you see any issue here.

Reply:
ok this look good when we have target branch (yml case), have you tried in case of classic editor, i think you won't get anything, just check once.

Reply:
Yes, I have checked. It is working as expected.